src/venue.njk

```diff
@@ -0,0 +1,131 @@
+                        <a href="https://www.google.com/maps/dir//WHVW%2B7RC+NIMHANS+Convention+Centre" 
+                           target="_blank" 
+                           class="inline-block bg-purple-600 text-white px-4 py-2 rounded-lg hover:bg-purple-700 transition duration-300">
+                            Find Directions
+                        </a>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="w-full overflow-hidden mb-6">
+            <iframe src="https://www.google.com/maps/embed?pb=!1m14!1m8!1m3!1d7776.895056958537!2d77.597019!3d12.94319!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x3bae15b78a61be57%3A0xc4498066da178922!2sNIMHANS%20Convention%20Centre!5e0!3m2!1sen!2sin!4v1740129911793!5m2!1sen!2sin" width="100%" class="rounded-lg" height="700" style="border:0;" allowfullscreen="" loading="lazy" referrerpolicy="no-referrer-when-downgrade"></iframe>
+        </div>
+
+        <div class="space-y-8">
+            <div class="bg-white shadow-lg rounded-xl p-8">
+                <h3 class="text-3xl font-bold text-black-900 mb-6">
+                    Transportation Options
+                </h3>
+                
+                <div class="grid md:grid-cols-2 gap-6">
+                    <div class="bg-purple-50 p-6 rounded-xl transform transition duration-300 hover:-translate-y-2 hover:shadow-md group flex flex-col">
+                        <div>
+                            <h4 class="text-2xl font-semibold text-purple-800 mb-4 group-hover:text-purple-700 transition duration-300">
+                                <i class="fas fa-subway mr-3 text-purple-600 group-hover:text-purple-500 transition duration-300"></i>Namma Metro
+                            </h4>
+                            <ul class="space-y-2 text-gray-700 flex-grow">
+                                <li>
+                                    <strong>From Green Line:</strong> Disembark at South End Circle or Lalbagh metro station. From the station, you can take a short taxi/auto ride to reach NIMHANS Convention Centre.
+                                </li>
+                                <li>
+                                    <strong>Approximate Distance:</strong> 2-3 km from metro stations
+                                </li>
+                                <li>
+                                    <strong>Estimated Travel Time:</strong> 10-15 minutes by auto/taxi
+                                </li>
+                            </ul>
+                        </div>
+                        <div class="mt-4 flex space-x-4">
+                            <a href="https://en.wikipedia.org/wiki/Green_Line_(Namma_Metro)" 
+                               target="_blank" 
+                               class="flex-1 text-center inline-block bg-purple-600 text-white px-4 py-2 rounded-lg hover:bg-purple-700">
+                                View Green Line Stations
+                            </a>
```

Comment:
Why is this a wikipedia link? We can use https://english.bmrc.co.in/schematic-route-map/

Reply:
Replaced (took wikipedia link from last year's website)